src/components/ModelDialog/components/ModalContent/styled.ts

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+export const StyledSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  margin: ${themeSpacing(4, 4, 14, 4)};
```

Comment:
14 also a magic number. But I guess it's the height of the footer.

Reply:
I've done the styling again, 'cause it was all a mess. Hopefully it is better now.